src/consts/tokens.yaml

```diff
@@ -0,0 +1,14 @@
+# A list of warp token configs
```

Comment:
I think we need to get a bit crisper IMO with our language. Very easy to add the warp token config that is input to the warp deploy command